tensorflow/core/kernels/sparse_fill_empty_rows_op.cc

```diff
@@ -185,41 +188,63 @@ struct SparseFillEmptyRows<CPUDevice, T, Tindex> {
 
 }  // namespace functor
 
+namespace {
+
+template <typename Device, typename T, typename Tindex>
+void SparseFillEmptyRowsOpImpl(
+    OpKernelContext* context, AsyncOpKernel::DoneCallback done = nullptr) {
+  if (!done) {
+    done = [] {};
+  }
```

Comment:
This could be a default value for `done` right?

Reply:
I actually did that originally but the CI failed with some bizarre compiler/linker errors. I couldn't reproduce it myself but guessed that this was the problem. I've added a comment to that effect.

Reply:
SG